broker/src/main/java/io/zeebe/broker/system/partitions/RaftMessagingService.java

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Camunda Services GmbH and/or licensed to Camunda Services GmbH under
+ * one or more contributor license agreements. See the NOTICE file distributed
+ * with this work for additional information regarding copyright ownership.
+ * Licensed under the Zeebe Community License 1.0. You may not use this file
+ * except in compliance with the Zeebe Community License 1.0.
+ */
+package io.zeebe.broker.system.partitions;
+
+import java.nio.ByteBuffer;
+import java.util.concurrent.Executor;
+import java.util.function.Consumer;
+
+/** Abstracts away messaging to other members of a given Raft - add operations as needed. */
+public interface RaftMessagingService {
+
+  /**
+   * Subscribes to a given subject - if another Raft member sends a message on this topic, the
```

Comment:
just member in the cluster - whether it is a raft member is more implementation specific or?

Reply:
It's another member in the same partition, I'll correct the naming